examples/default.yaml

```diff
@@ -208,9 +208,11 @@ firmware:
   legacyBIOS: null
 
 video:
-  # QEMU display, e.g., "none", "cocoa", "sdl", "gtk".
+  # QEMU display, e.g., "none", "cocoa", "sdl", "gtk", "default".
   # As of QEMU v6.2, enabling this is known to have negative impact
   # on performance on macOS hosts: https://gitlab.com/qemu-project/qemu/-/issues/334
+  # Choosing "none" will hide the video output, and not show any window.
+  # Choosing "default" will pick the first available of: gtk, sdl, cocoa.
```

Comment:
It should be noted that this is the default value of QEMU, not Lima

Reply:
Hmm, that was supposed to be on the line directly below that one.
```
  # 🟢 Builtin default: "none"
```

Reply:
I guess that are confusing to new comers and need explicit explanation

Reply:
Maybe it would be less confusing, if all five alternatives were explained ?

Reply:
Perhaps if the text is not close to the other one, it would be better ? Flipped them...